api/src/usersAvatar/services/usersAvatar.service.ts

```diff
@@ -45,8 +62,8 @@ export class UsersAvatarService {
             throw new InternalServerErrorException(err.message);
         }
 
-        await this.usersAvatarRepository.delete(id).catch(err => {
-            throw new InternalServerErrorException(err);
+        await this.usersAvatarRepository.delete(id).catch(error => {
+            throw error
```

Comment:
Missing semicolon

Reply:
Yeah, i must configure prettier to run on save 😁

Reply:
done